CMakeLists.txt

```diff
@@ -19,15 +46,41 @@ if(NOT CMAKE_BUILD_TYPE)
     set (CMAKE_BUILD_TYPE  "RelWithDebInfo" CACHE STRING "Choose the type of build, options are: None(CMAKE_CXX_FLAGS or CMAKE_C_FLAGS used) Debug Release RelWithDebInfo MinSizeRel." FORCE)
 endif(NOT CMAKE_BUILD_TYPE)
 
-include_directories (inc)
-include_directories (lib)
 
-# Application modules
+find_package (Boost COMPONENTS filesystem iostreams program_options regex log thread system REQUIRED )
+include_directories (${Boost_INCLUDE_DIR} )
+find_package (Threads REQUIRED)
+set (ODB_PATH "/usr/local/lib" CACHE STRING "Absolute path to ODB libraries installation folder.")
 
-set (MOD_ODB_SOURCES)
 
-set (ODB_MAPPING_CLASSES inc/vcf/error.hpp)
 
+# APPLICATION MODULES
+add_library(mod_sqlite3 STATIC lib/sqlite/sqlite3.c)
+
+if (BUILD_STATIC)
+set (LIBRARIES_TO_LINK_TO_MOD
```

Comment:
Maybe `THIRD_PARTY_LIBRARIES` would be a slightly nicer name? Because `MOD` is just a generic prefix that would become ambiguous if we had 2 modules/libraries.